classes/Risk.php

```diff
@@ -63,16 +71,25 @@ public function getFields()
     public function getTranslationsFieldsChild()
     {
         $this->validateFieldsLang();
-        return $this->getTranslationsFields(array('name'));
+
+        return $this->getFieldsLang();
```

Comment:
@julienbourdeau / @thierrymarianne / @maximebiloe / @mickaelandrieu
Another of the core team to confirm that it is ok?